### PR TITLE
Review slide edits

### PR DESCRIPTION
**Pull request by MCLusardi**

### CODE REVIEW

File: _slides/lovable_linux.md [hunk 1]
```diff
@@ -98,3 +98,201 @@ Swap File - specific file on disk that the pages get written to. Another solutio
...
+Briefly explain permission bits (including sticky and setuid bits) for files and directories.
+
+<vertical />
+
+```drwxrwxr-x```
+
+```d``` or ```-``` : directory or file
+```rwx``` : What the owner of the file is allowed to do (read, write, execute)
+```rwx```: What users in the owner's group are allowed to do (read, write, execute)
+```r-x```: What everyone else is allowed to od (read and execute, but not write)
```
Comment (RMenezes782) [inline _slides/lovable_linux.md]: *do

File: _slides/lovable_linux.md [hunk 2]
```diff
@@ -274,25 +288,55 @@ Describe the services provided by TCP but not UDP.
...
 
 <horizontal />
 
-File System
\ No newline at end of file
+## File System
+
+<vertical />
+
+*What information is stored in an i-node? What file system information is not?*
```
Comment (RMenezes782) [inline _slides/lovable_linux.md:323]: *inode

**Review by RMenezes782 — COMMENTED** [review]
Left a few comments, mostly nits but a few enhancements on definitions.

File: _slides/lovable_linux.md [hunk 3]
```diff
@@ -3,48 +3,98 @@ layout: slide
...
 
 <vertical />
 
-2.5 What is a page fault? When is it an error? When is it not an error?
+What is a page fault? When is it an error? When is it not an error?
 
 <vertical />
 
-* Page fault – When a running program tries to access some virtual memory in its address space that is not mapped to physical memory. There are three types
+Page fault – program attempts access to virtual memory that is not mapped to physical memory.
```
[the remark this answers is not in the document]
Comment (MCLusardi) [reply _slides/lovable_linux.md:68]: How about "attempts access to virtual memory not mapped to up-to-date physical memory"

File: _slides/lovable_linux.md [hunk 4]
```diff
@@ -3,48 +3,98 @@ layout: slide
...
+When it’s not an Error - Major or Minor
 
 <vertical />
 
-2.6 What is Spatial and Temporal Locality? Swapping? Swap file? Demand Paging? 
+What is Spatial and Temporal Locality? Swapping? Swap file? Demand Paging? 
+
+<vertical />
+
+Spatial Locality - Objects in adjacent memory addresses get used (think arrays). That is why a page table is a certain size.
```
[the remark this answers is not in the document]
Comment (MCLusardi) [reply _slides/lovable_linux.md:90]: I believe it should be page. Virtual addresses correspond to pages, not tables.

And no, I don't think the TLB benefits from both. The TLB provides direct virtual to physical address translation. If you have another virtual address that exists in the same page as a TLB entry, but is not a TLB entry itself, you still have to go through the whole process of searching through page tables.

File: _slides/lovable_linux.md [hunk 5]
```diff
@@ -98,3 +98,201 @@ Swap File - specific file on disk that the pages get written to. Another solutio
...
+
+<vertical />
+
+### Coffman Conditions
+
+Circular wait: There exists a cycle in the Resource Allocation graph
+
+* P1 is waiting for resources from P2
+* P2 is waiting for resources from P3
+* ect...
```
[the remark this answers is not in the document]
Comment (MCLusardi) [reply _slides/lovable_linux.md:212]: The loop can involve any number of processes. P3's waiting for P4, P4's waiting for P5, P5's waiting for P6, ... You get the idea

**Pull request merged**